### PR TITLE
MOSIP-27896 - [MOSIP-23945] Bean definition modified, [MOSIP-27987] Updated prereg db_upgrade_scripts

### PR DESCRIPTION
**Pull request by aihamh**

### CODE REVIEW

**Review by mayuradesh — APPROVED** [review]

File: db_upgrade_scripts/mosip_prereg/ddl/prereg-otp_transaction.sql [hunk 1]
```diff
@@ -1,63 +0,0 @@
--- -------------------------------------------------------------------------------------------------
```
Comment (vishwa-vyom) [inline db_upgrade_scripts/mosip_prereg/ddl/prereg-otp_transaction.sql:1]: why is this file deleted from upgrade ?

File: db_upgrade_scripts/mosip_prereg/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql [hunk 2]
```diff
@@ -16,6 +16,15 @@ GRANT ALL PRIVILEGES ON ALL TABLES IN SCHEMA prereg TO postgres;
 
 ALTER TABLE prereg.reg_appointment DROP CONSTRAINT IF EXISTS fk_rappmnt_id CASCADE;
 
+CREATE INDEX IF NOT EXISTS idx_app_demo_cr_by ON prereg.applicant_demographic USING btree cr_by COLLATE pg_catalog."default" ASC NULLS LAST) TABLESPACE pg_default;
```
Comment (vishwa-vyom) [inline db_upgrade_scripts/mosip_prereg/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql]: is bracket missing here ?

**Review by vishwa-vyom — CHANGES_REQUESTED** [review]

File: db_upgrade_scripts/mosip_prereg/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql (hunk 2 above)
Comment (aihamh) [reply db_upgrade_scripts/mosip_prereg/sql/1.1.5.5_to_1.2.0.1-B1_upgrade.sql]: Added missing bracket

File: db_upgrade_scripts/mosip_prereg/ddl/prereg-otp_transaction.sql (hunk 1 above)
Comment (aihamh) [reply db_upgrade_scripts/mosip_prereg/ddl/prereg-otp_transaction.sql:1]: “prereg-otp_transaction.sql"  is NOT used in 1.1.5.5_to_1.2.0.1-B1_upgrade.sql script

**Review by vishwa-vyom — APPROVED** [review]
